CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ This project tries to follow [SemVer 2.0.0](https://semver.org/).
 
 ## v1.2.0 (2021-07-12)
 
+- Added support for the TZ environment variable (setting timezones ex.
+  `"Europe/Stockholm"`) through the tzdata package. (#20)
+
```

Comment:
Should be put in a new entry with `(WIP)` for the date.
I just merged #14 with a v1.3.0 entry.

Reply:
Good note!